### PR TITLE
Mixin update for usercheckpoint

### PR DESCRIPTION
**Pull request by vdanix**
Mixin update for usercheckpoint patch

Change-Id: I4ac9ce24a6e584eda355f8c3c2750203d43b5faa
Tracked-On: OAM-97424
Signed-off-by: vdanix <vishwanathx.dani@intel.com>

### CODE REVIEW

**Comment by vdanix** [comment]
Hi,
Kindly review the PR,

@JeevakaPrabu @sgnanase m.balaji@intel.com

**Review by sgnanase — APPROVED** [review]

File: caas/overlay/frameworks/base/core/res/res/values/config.xml [hunk 1]
```diff
@@ -197,4 +197,8 @@
         <item>bugreport</item>
         <item>users</item>
     </string-array>
+    <!-- Names of packages that should not be suspended when personal use is blocked by policy. -->
+    <string-array name="config_packagesExemptFromSuspension" translatable="false">
+        <item>com.google.android.apps.wellbeing</item>
+    </string-array>
```
Comment (JeevakaPrabu) [inline caas/overlay/frameworks/base/core/res/res/values/config.xml:203]: Why is this change part of usercheckpoint patch?

File: caas/extra_files/codecs/media_codecs_gen9.xml [hunk 2]
```diff
@@ -78,7 +78,6 @@ Only the three quirks included above are recognized at this point:
     <Decoders>
 
 
-
         <MediaCodec name="OMX.Intel.hw_vd.h264" type="video/avc" >
```
Comment (JeevakaPrabu) [inline caas/extra_files/codecs/media_codecs_gen9.xml:81]: Why is line removal change part of this patch?

File: caas/extra_files/codecs/mfx_omxil_core.conf [hunk 3]
```diff
@@ -2,7 +2,6 @@
 OMX.Intel.hw_vd.h264 : libmfx_omx_components_hw.so
 
 
-
 OMX.Intel.hw_ve.h264 : libmfx_omx_components_hw.so
```
Comment (JeevakaPrabu) [inline caas/extra_files/codecs/mfx_omxil_core.conf:5]: Why is line removal change part of this patch?

File: celadon_tablet/extra_files/codecs/mfx_omxil_core.conf [hunk 4]
```diff
@@ -2,7 +2,6 @@
 OMX.Intel.hw_vd.h264 : libmfx_omx_components_hw.so
 
 
-
 OMX.Intel.hw_ve.h264 : libmfx_omx_components_hw.so
```
Comment (JeevakaPrabu) [inline celadon_tablet/extra_files/codecs/mfx_omxil_core.conf:5]: Why is line removal change part of this patch?

**Review by JeevakaPrabu — CHANGES_REQUESTED** [review]

**Pull request closed without merging**